src/test/java/task_at_7/CalculatorTest.java

```diff
@@ -0,0 +1,39 @@
+package task_at_7;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CalculatorTest {
+
+
+    @Test
+    public void mathOperationAdd() {
+        Calculator calculator = new Calculator();
+        Assert.assertEquals(10, calculator.mathOperationAdd(5, 5));
```

Comment:
based on which technique do you select this values as representative for this scenario?

Reply:
It based on EP. This values are in valid range of positive testing. But it`s not enough of them to properly test the method.